modules/gapi/src/backends/cpu/gcpuimgproc.cpp

```diff
@@ -180,6 +180,26 @@ GAPI_OCV_KERNEL(GCPUCanny, cv::gapi::imgproc::GCanny)
     }
 };
 
+GAPI_OCV_KERNEL(GCPUCalcOptFlowLK, cv::gapi::imgproc::GCalcOptFlowLK)
```

Comment:
All stuff you've added here should be moved to the separate file which is dedicated for G-API kernels on CPU for OpenCV video module functions: modules/gapi/src/backends/cpu/gcpuvideo.cpp (which you should create).